packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -4,204 +4,549 @@
 
 import 'dart:math';
 
+import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter/widgets.dart';
 
-import '../../diagnostics_node.dart';
+import '../../../ui/colors.dart';
+import '../../../ui/theme.dart';
 import '../inspector_data_models.dart';
 import 'arrow.dart';
 import 'utils.dart';
 
-@immutable
-class StoryOfYourFlexWidget extends StatelessWidget {
-  const StoryOfYourFlexWidget({
-    this.diagnostic,
-    this.properties,
+const widthIndicatorColor = mainUiColor;
+const heightIndicatorColor = mainGpuColor;
+
+String getCrossAxisAssetImageUrl(CrossAxisAlignment alignment) {
+  return 'assets/img/story_of_layout/cross_axis_alignment/${describeEnum(alignment)}.png';
+}
+
+String getMainAxisAssetImageUrl(MainAxisAlignment alignment) {
+  return 'assets/img/story_of_layout/main_axis_alignment/${describeEnum(alignment)}.png';
+}
+
+class StoryOfYourFlexWidget extends StatefulWidget {
+  const StoryOfYourFlexWidget(
+    this.properties, {
     Key key,
-  }) : super(key: key);
+  })  : assert(properties != null),
+        super(key: key);
+
+  final FlexLayoutProperties properties;
+
+  @override
+  _StoryOfYourFlexWidgetState createState() => _StoryOfYourFlexWidgetState();
+}
+
+class _StoryOfYourFlexWidgetState extends State<StoryOfYourFlexWidget> {
+  static const kBottomHeight = 16.0;
```

Comment:
dart style: remove the "k"

Reply:
I use the "k" here because I saw some constants in Flutter core framework following the same convention (`flutter/lib/src/material/constants.dart`)
But yes sure!

Reply:
yeah that Flutter code unfortunately is inconsistent with the style guide and it would be a big breaking change to modify it at this point.
https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names

Reply:
just knew that! Thanks for pointing that out Jacob.